abap_transpile.json

```
@@ -30,110 +30,92 @@
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "too_many_enum_values"},
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "too_many_number_annotations"},
       {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "too_many_required_annotations"},
       {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "too_many_titles_and_showalways"},
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "too_many_value_links", "note": "??"},
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "wrong_usage_callback_class"},
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "wrong_usage_default"},
-      {"object": "ZCL_AFF_ABAP_DOC_PARSER", "class": "ltcl_aff_abap_doc_parser", "method": "wrong_usage_enum_values"},
       {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "call_reader_and_decode", "note": "Void type: cl_oo_abap_doc_reader"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "callback_class_is_invalid", "note": "??"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "callback_class_is_valid", "note": "cl_oo_classname_service=>get_all_method_includes"},
       {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_abap_doc_for_absolute_name", "note": "Void type: cl_oo_abap_doc_reader"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_struc_of_enum_values_cl"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_numeric", "note": "abap.types.typeTodoDecFloat16Type is not a constructor"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_string_like"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "name_mapping_and_camel_case", "note": "VALUE #( table_expr OPTIONAL )"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "name_mapping_found", "note": "VALUE #( table_expr OPTIONAL )"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "stack_stores_operations", "note": "VALUE #( stack[ 1 ]-operation OPTIONAL )"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "validate_default", "note": "abap.types.typeTodoDecFloat16Type is not a constructor"},
-      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "value_mapping_found"},
+      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "stack_stores_operations", "note": "??"},
+      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_string_like_enum", "note": "??"},
```

Doest it make sense to add a note here like 
```suggestion
      {"object": "ZCL_AFF_WRITER", "class": "ltcl_type_writer", "method": "get_type_info_string_like_enum", "note": "enums are not yet supported"},
```